src/clojure_lsp/crawler.clj

```diff
@@ -216,5 +219,8 @@
         (stubs/generate-and-analyze-stubs! settings db))
       (producer/publish-progress (:producer @db) 95 "Analyzing project files" progress-token)
       (log/info "Analyzing source paths for project root" root-path)
-      (analyze-source-paths! (:source-paths settings) db))
+      (analyze-source-paths! (:source-paths settings) db)
+      (producer/publish-progress (:producer @db) 97 "Analyzing test files" progress-token)
```

Comment:
I think we should check for `when-not (:api? @db)` and we should not this on this same thread or we will block the clojure-lsp startup time, maybe just move this to [here](https://github.com/clojure-lsp/clojure-lsp/pull/698/files#diff-fa419531b6cbafb2899aa71bcd5a5acee775a79480a43235c6f29fc5074cb1b6R215) would solve both issues, WDYT?

Reply:
> we should not this on this same thread or we will block the clojure-lsp startup time

I just realized there is a already a go block inside the producer call, so it's fine